target/linux/lantiq/image/Makefile

```diff
@@ -714,6 +729,30 @@ define Device/arcadyan_vgv7519-nor
 endef
 TARGET_DEVICES += arcadyan_vgv7519-nor
 
+define Device/arcadyan_vgv952cjw33-e-ir
+  $(Device/NAND)
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := VGV952CJW33-E-IR
+  DEVICE_ALT0_VENDOR := Vodafone
+  DEVICE_ALT0_MODEL := Easybox 904xDSL
+  DEVICE_DTS := VGV952CJW33-E-IR
+  SUPPORTED_DEVICES += VGV952CJW33-E-IR
+  IMAGES := sysupgrade.bin
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | lzma | uImage lzma | pad-offset 16 0 | append-uImage-dummyrootfs
+endef
+
+define Device/arcadyan_vgv952cjw33-e-ir-vpe
+  $(Device/arcadyan_vgv952cjw33-e-ir)
+  DEVICE_VARIANT := VPE
+  DEVICE_ALT0_VARIANT := vpe
+  DEVICE_DTS := VGV952CJW33-E-IR
+  DEVICE_PACKAGES := kmod-usb-dwc2 kmod-ltq-tapi kmod-ltq-vmmc wpad-basic
+  SUPPORTED_DEVICES += VGV952CJW33-E-IR
```

Comment:
This can be dropped. SUPPORTED_DEVICES is derived from the compatible in DTS.

To make that work for your two devices, you will need to have two DTS files, one for each device, which then both include a common DTSI file containing most of the code. Again, have a look at how this is done for the fritz3370, just think away the &localbus in DTS files there. So, you would have

vr9_arcadyan_vgv952cjw33-e-ir.dtsi
vr9_arcadyan_vgv952cjw33-e-ir-vpe.dts
vr9_arcadyan_vgv952cjw33-e-ir-smp.dts

Then, with my latest patch you would just need to specify "SOC := vr9" here and the DTS name would be calculated automatically. Despite, by putting the correct compatible into each DTS, you get rid of the SUPPORTED_DEVICES line.